app/src/main/java/org/mozilla/focus/home/HomeFragment.kt

```diff
@@ -259,9 +266,27 @@ private fun onBindCustomHomeTile(uiLifecycleCancelJob: Job, holder: TileViewHold
         }
 
         // Wait for both to complete so we can animate them together.
-        iconView.setImageBitmap(screenshot.await()) // TODO: if null, provide placeholder.
+        val screenshotToRound = screenshot.await() // TODO: if null, provide placeholder.
+        val cornerRadius = 8.toFloat()
+        // Need copy of screenshot bitmap because it is immutable
+        val copyOfScreenshot = Bitmap.createBitmap(screenshotToRound.width, screenshotToRound.height, screenshotToRound.config)
+        val canvas = Canvas(copyOfScreenshot)
+        val paint = Paint()
+        // Need to set isAntiAlias to true because it smooths out the edges of what is being drawn
+        paint.isAntiAlias = true
+        // Need the shader to replicate the edge color if the shader draws outside of its original bounds
+        paint.shader = BitmapShader(screenshotToRound, Shader.TileMode.CLAMP, Shader.TileMode.CLAMP)
+        canvas.drawRoundRect(RectF(0.0f, 0.0f, screenshotToRound.width.toFloat(), screenshotToRound.height.toFloat()), cornerRadius, cornerRadius, paint)
+
+        iconView.setImageBitmap(copyOfScreenshot)
         titleView.text = title.await()
 
+        val layoutParams = LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT, 130)
```

Comment:
RecyclerView re-uses views so if you change these layout parameters, you also have to change them in `onBindBundled` (since the views could be re-used for bundle tiles.

You should remove these layout params from the XML, add a comment mentioning they're defined in code, and then define them in both `onBind*` methods (or perhaps define them outside, perhaps inside the when, so that we're less likely to forget to update both).